lib/src/rules/annotate_overrides.dart

```diff
@@ -78,7 +85,7 @@ class _Visitor extends SimpleAstVisitor<void> {
   }
 
   Element? getOverriddenMember(Element member) {
-    var classElement = member.thisOrAncestorOfType<ClassElement>();
+    var classElement = this.classElement;
```

Comment:
The one possible issue here is that we can't clear the field when we exit a class. So if we visit a class and then visit an extension we'll use the element from the previous class. We can address this by visiting `ExtensionDeclaration`s and `MixinDeclaration`s. I can't think of any other containers that contain fields or methods.

Reply:
Ah.  Good catch.

Reply:
Fixed.